lib/uff_db_loader.rb

```diff
@@ -125,6 +125,12 @@ def create_initializer
 
     private
 
+    def container_name(environment)
+      return "#{config.app_name}_#{environment}_db" if config.container_name_fn.blank?
```

Comment:
Wonder if we should allow this to be a string too.

```rb
return "#{config.app_name}_#{environment}_db" if config.container_name_fn.blank?
return config.container_name.fn unless config.container_name_fn.respond_to? :call

config.container_name_fn.call(config.app_name, environment)
```

This would be the most flexible way IHMO. Name of the config setting should be different then.

Reply:
Ah yes! I had something similar before but not in the same config option. I like it.

Reply:
Hmm. On second thought that would only help if you had the same container name for all environments.

Reply:
Which can be a use case (e.g. one database container for all environments with different database names though)

Reply:
I guess. I did implement the option to have a string yesterday, so I guess I'll just merge now? Do you want to join the discussion before I do that @malfynnction?